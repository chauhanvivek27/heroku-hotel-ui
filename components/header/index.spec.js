import React from "react";
import { mount } from "enzyme";
import { renderToJson } from "enzyme-to-json";
import Header from "./index";

describe("Header Render test", () => {
  let wrapper;

  it("should render correctly", () => {
    wrapper = mount(<Header />);
    expect(renderToJson(wrapper.render())).toMatchSnapshot();
  });
});
