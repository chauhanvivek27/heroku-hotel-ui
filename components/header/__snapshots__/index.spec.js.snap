// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Render test should render correctly 1`] = `
<div
  class="header"
>
  <h1
    class="headertitle"
  >
    Hotel Search
  </h1>
</div>
`;
